frontend/components/Domain/Recipe/RecipeExplorerPage.vue

```diff
@@ -67,12 +67,16 @@
               <v-list>
                 <v-list-item @click="toggleOrderDirection()">
                   <v-icon left>
-                    {{ $globals.icons.sort }}
+                    {{
+                      state.orderDirection === "asc" ?
+                      $globals.icons.sortAscending : $globals.icons.sortDescending
+                    }}
```

Comment:
The icons should be the other way around. 
Tho, i have to say, looking at this i have a kinda hard time making sense of the icon. Why is the ascending icon having an arrow down? 🧐 Especially because the calendar sorting icon used on the timeline does it the other way. 

```suggestion
                    {{
                      state.orderDirection === "asc" ?
                      $globals.icons.sortDescending : $globals.icons.sortAscending
                    }}
```

Reply:
Should they be the other way around? The way I understand it is that it's showing the current state, and you want to click on it to reverse the state. In which case it should match:

![image](https://github.com/mealie-recipes/mealie/assets/71845777/2b549db1-e23b-47bd-8344-86d15080f0c7)

Reply:
The current state is 'Descending' right? 

So having the descending icon beside 'Sort Ascending' is confusing (at least to me)

Reply:
Oh FML I confused myself. You are correct, we are currently descending, so we should flip the icon. I was thinking we were ascending

Reply:
Those icons are absolutely confusing 😅